src/js/components/search/topFilterBar/filterGroups/NAICSFilterGroup.jsx

```diff
@@ -37,23 +37,13 @@ export default class NAICSFilterGroup extends React.Component {
 
     generateTags() {
         return this.props.filter.values
-            .map((naics) => {
-                if (naics.isV2) {
-                    return {
-                        value: `${naics.identifier}`,
-                        title: `${naics.value} - ${naics.label} (${naics.count})`,
-                        isSpecial: false,
-                        // doesn't appear to be being used...
-                        removeFilter: () => this.removeFilter
-                    };
-                }
-                return {
-                    value: `${naics.identifier}`,
-                    title: `${naics.naics} | ${naics.naics_description}`,
-                    isSpecial: false,
-                    removeFilter: this.removeFilter
-                };
-            });
+            .map((naics) => ({
+                value: `${naics.identifier}`,
+                title: `${naics.value} - ${naics.label} (${naics.count})`,
+                // doesn't appear to be being used...
+                isSpecial: false,
```

Comment:
Agree, we can go ahead and remove the `isSpecial` prop

Reply:
199ee87b2


☝️ touches 24 files 😱 .

All of em though just call the same component. Seems like a really inefficient way of rendering this markup!